README.md

```diff
@@ -1418,6 +1418,35 @@ Other Style Guides
     }
     ```
 
+    <a name="variables--unary-increment-decrement"></a><a name="13.5"></a>
```

Comment:
there's no need to add a link to "13.5" - the numbers only exist for legacy reasons.